modules/gapi/test/common/gapi_video_tests_common.hpp

```diff
@@ -321,6 +322,31 @@ inline GComputation runOCVnGAPIOptFlowPipeline(TestFunctional& testInst,
     return c;
 }
 
+inline void testBackgroundSubtractorStreaming(cv::GStreamingCompiled gapiBackSub,
+    cv::BackgroundSubtractor* pOCVBackSub,
```

Comment:
This parameter should be `shared_ptr` or `cv::Ptr`

Reply:
Done